content/reference/sql/server/replication.md

```diff
@@ -4,32 +4,45 @@ title: Replication
 
 # Replication
 
-Dolt supports two forms of
-[replication](../../../concepts/dolt/rdbms/replication.md).
+Dolt supports two forms of [replication](../../../concepts/dolt/rdbms/replication.md) where a Dolt primary sql-server replicates changes to Dolt sql-server replicas. If you're looking to replicate changes from a MySQL primary server to a Dolt replica, check out the guide to [MySQL Replication to Dolt](../../../guides/binlog-replication.md). 
 
-In one, Dolt uses a remote as a middleman to facilitate replication between the
+In **Remote-Based Replication**, Dolt uses a remote as a middleman to facilitate replication between the
 primary and read replicas. In this mode, Dolt replication triggers on a [Dolt
 commit](../../../concepts/dolt/git/commits.md).
 
 ![Read replication](../../../.gitbook/assets/dolt-read-replication.png)
 
-In the second, the primary dolt sql-server instance replicates all writes to a
+This is the simplest form of replication to configure and administer. Use this form of replication
+when you do **not** need the hot-standby support of Direct-to-Standby Replication. See
+[Direct vs Remote Replication](replication.md#direct-vs.-remote-replication) for more
+details on the differences between Remote-Based Replication and Hot Standby Replication.
+
+
+In **Direct-to-Standby Replication**, the primary dolt sql-server instance replicates all writes to a
 set of configured standby servers. In this mode, there is no intermediate
 [remote](../../../concepts/dolt/git/remotes.md) and all SQL transaction commits
 are replicated, not just Dolt commits.
 
 ![Standby replication](../../../.gitbook/assets/dolt-standby-replication.png)
 
-This page describes configuration and considerations for both types of
+Use this form of replication when you have high-availability requirements, need a hot standby
+server ready to swap in for the primary, and you do **not** need to replicate from a MySQL primary
```

Comment:
I wouldn't mention MySQL at all here, make its own section

Reply:
Whoops! Good catch. I meant to take that part out after I moved the binlog content out and into a guide. Will fix.